### ISSUE TITLE
npm install fetch failed with status code 404

### ISSUE THREAD

**Issue by cocacrave** [topdmc/react-chartjs2#2]
I'm getting error when trying to npm install --save react-chartjs2
npm version 3.6.0
node version 5.7.1

```
→ npm install --save react-chartjs2
npm ERR! fetch failed https://registry.npmjs.org/react-chartjs2/-/react-chartjs2-1.0.1.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
npm ERR! fetch failed https://registry.npmjs.org/react-chartjs2/-/react-chartjs2-1.0.1.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
npm ERR! fetch failed https://registry.npmjs.org/react-chartjs2/-/react-chartjs2-1.0.1.tgz
npm ERR! Darwin 15.5.0
npm ERR! argv "/Users/cocacrave/.nvm/versions/node/v5.7.1/bin/node" "/Users/cocacrave/.nvm/versions/node/v5.7.1/bin/npm" "install" "--save" "react-chartjs2"
npm ERR! node v5.7.1
npm ERR! npm  v3.6.0

npm ERR! fetch failed with status code 404
npm ERR! 
```

**Comment by chriskinsman** [comment]
I am seeing something similar with a status code of 400

**Comment by houjiazong** [comment]
I published [v1.0.2](https://github.com/topdmc/react-chartjs2/releases/tag/v1.0.2), Please check your network, try again
